env

```diff
@@ -0,0 +1,125 @@
+#--------------------------------------------------------------------
```

Comment:
I think you should exclude the `.env` file incase it messes with other people's work when they merge